src/clj/chronograph/domain/organization.clj

```diff
@@ -12,3 +12,13 @@
                        :organization-id id
                        :role acl/admin})
       organization)))
+
+(defn find-if-authorized
```

Comment:
This could be refactored into a generic macro.

Reply:
Punting. It's not clear what generic case the macro should address. Access to resources depends on how we choose to deal with role-based authorization, which in turn is not well defined.